crates/cadenza-eval/test-data/block-simple.cdz

```diff
@@ -0,0 +1,5 @@
+let foo =
```

Comment:
we also want to make sure this works with functions as well:
```
fn foo a b =
    let av = a * 2
    let bv = b * 3
    av * bv
```

additionally, we should make sure it nests:

```
let foo =
    let bar =
        let baz =
            1
        baz
    bar
foo
```